README.md

```diff
@@ -18,14 +18,34 @@ People who viewed this item also viewed
 1. [Development](#development)
 
 ## Setup
+
+Create .env file in root folder of your project and specify following
+// set port number for express server
+PORT=3004
+
+// configure database
+MYSQL_DB_HOST=localhost
+MYSQL_DB_USERNAME=root
+MYSQL_DB_PASSWORD=
+MYSQL_DB_DATABASE=alsoviewed
+
+// specify total number of items to seed, with their start id, end id
+TOTAL_ITEMS=100
```

Comment:
Isn't this redundant, since you're specifying a start id and end id? (total = end id - start id)

Reply:
In that case total will be less than one(10 - 1 = 9). I am using this variable for testing purpose only. Secondly it seems redundant but it keeps clarity of what you aiming for instead of referring to ids which can be odd in some cases.